### PR TITLE
[nGraph] VisualizeTree displays tensors

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *VisualizeTree displays tensors*

### CODE REVIEW

File: src/core/src/pass/visualize_tree.cpp [hunk 1]
```diff
@@ -505,6 +505,44 @@ string pass::VisualizeTree::get_node_name(shared_ptr<Node> node) {
...
+            size_t i = 0;
+            for (const auto name : names) {
+                ss << (i == 0 ? "" : ", ") << name;
+                i++;
+            }
+            return ss.str();
+        };
+
+        if (node->get_input_size() != 0) {
+            rc += "\\n" + (nvtmn ? string("in_tensor_names: ") : "");
```
Comment (itikhono) [inline src/core/src/pass/visualize_tree.cpp:521]: I believe it's better to use `stringstream` instead of `string` for `rc` variable

**Review by itikhono — APPROVED** [review]

**Review by v-Golubev — APPROVED** [review]
In general LGTM. Could you please also update the corresponding [readme](https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/model_representation.md?plain=1#L164) file?

**Comment by eshoguli** [comment]
> In general LGTM. Could you please also update the corresponding [readme](https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/model_representation.md?plain=1#L164) file?

thanks, will be done later

**Comment by ilya-lavrenov** [comment]
Please, fix https://dev.azure.com/openvinoci/dldt/_build/results?buildId=508194&view=logs&j=e0e676f4-9664-538a-6d6b-aa53516aa09f&t=839cb6ab-4ea8-51d4-e102-9a106ad673bf&l=2510

Compiler is clang

**Pull request merged**